### ISSUE TITLE
Request: {{ import }} in vento

### ISSUE THREAD

**Issue by tjaynl** [oscarotero/vento#9]
So i was wondering if there is a {{ import "file" as file }} option in Vento and if not would it be possible to request this feature for a future release?

**Comment by oscarotero** [comment]
What's the use case of this?
It's possible to import javascript modules, but I assume you're talking about something different.

**Comment by tjaynl** [comment]
So i want to import some macro's that could be used in a markdown file for example i have a macro.njk which contains some macro's like:

```
{% macro even_columns(left, right, classes) %}
<section class="even-columns {{ classes }}">
    <img src="{{ left }}" alt="">
    <img src="{{ right }}" alt="">
</section>
{% endmacro %}
```

i want to import this file so the macro's can be used when writting content.

**Comment by oscarotero** [comment]
Ok, understood. It's not possible right now, but it's a nice to have feature. I'll work on that.

**Comment by oscarotero** [comment]
Implemented in v0.7.0
- [Functions](https://vento.js.org/2-API/function/)
- [Import and export tags](https://vento.js.org/2-API/import-export/)
